### ISSUE TITLE
Retrieve playlist containing a video

### ISSUE THREAD

**Issue by Vrihub** [mps-youtube/mps-youtube#534]
Feature request (I hope the youtube API allows to do that): add a command to retrieve the playlist that contains a video. E.g. `l <number>` (`l` obviously stands for `list`) would return the playlist (if existing) that contains the video `<number>`.

At the moment the `r` command is a workaround for this, but the results suggested by youtube are not always optimal. And also searching for the video in playlists with the `//` command is not always convenient.

**Comment by ids1024** [comment]
But a video could be in multiple playlists, right?

**Comment by Vrihub** [comment]
> But a video could be in multiple playlists, right?

Right. I forgot to add: the playlist should be the one belonging to the user that uploaded the video

**Comment by Vrihub** [comment]
Alternatively, would it be possible to do a search combining both search terms and a username?
E.g.: search for videos containing terms X and Y, uploaded by user Z
Is this allowed by the youtube API?

**Comment by ids1024** [comment]
Mpsyt already allows searching a users videos.

Try `user rushvevo/limelight`, for example.
